app/mailers/user_mailer.rb

```diff
@@ -1,6 +1,11 @@
 class UserMailer < ApplicationMailer
   default from: "Epic Logger <admin@epiclogger.com>"
 
+  def email_confirmation(token)
```

Comment:
this seems wrong and it should not be done this way.
The way to call emails is by doing this with action_mailer: `UserMailer.email_confirmation(user).deliver_later` so notice there is the full user record sent. That's how this should work as well. So the only thing you would do in this method would be a `@user = user` to assign the variable for mailer view.